scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/Cli.kt

```diff
@@ -38,6 +38,7 @@ object Cli {
     val logLevelKafka = Key("log.level.kafka", enumType(*Level.values()))
     val namespace = Key("pod.namespace", stringType)
     val dataflowReplicaId = Key("dataflow.replica.id", stringType)
+    val dispatcherNThreads = Key("dispatcher.n.threads", intType)
```

Comment:
I think that this setting should have a more end-user friendly name, relating to what it does rather than to implementation details. I suggest something like `pipeline.ctlops.threads`